be/src/storage/lake/tablet_manager.h

```diff
@@ -43,14 +46,20 @@ class TabletManager {
     Status put_txn_log(const std::string& group, TxnLogPtr log);
     StatusOr<TxnLogPtr> get_txn_log(const std::string& group, int64_t tablet_id, int64_t txn_id);
     Status delete_txn_log(const std::string& group, int64_t tablet_id, int64_t txn_id);
+    StatusOr<TabletMetaIter> list_tablet_metadata(const std::string& group);
+    StatusOr<TabletMetaIter> list_tablet_metadata(const std::string& group, int64_t tablet_id);
+    StatusOr<TxnLogIter> list_txn_log(const std::string& group);
+    StatusOr<TxnLogIter> list_txn_log(const std::string& group, int64_t tablet_id);
+    StatusOr<TabletMetadataPtr> get_tablet_metadata(const std::string& group, const std::string& meta_path);
+    StatusOr<TxnLogPtr> get_txn_log(const std::string& group, const std::string& txnlog_path);
 
 private:
     std::string tablet_meta_path(const std::string& group, int64_t tablet_id, int64_t verson);
+    std::string tablet_meta_path(const std::string& group, const std::string& meta_path);
     std::string txn_log_path(const std::string& group, int64_t tablet_id, int64_t txn_id);
-    StatusOr<MetadataIterator> list_tablet_metadata(const std::string& group);
-    StatusOr<MetadataIterator> list_tablet_metadata(const std::string& group, int64_t tablet_id);
-    StatusOr<MetadataIterator> list_txn_log(const std::string& group);
-    StatusOr<MetadataIterator> list_txn_log(const std::string& group, int64_t tablet_id);
+    std::string txn_log_path(const std::string& group, const std::string& txnlog_path);
+    StatusOr<TabletMetadataPtr> _get_tablet_metadata(const std::string& meta_full_path);
```

Comment:
```suggestion
    StatusOr<TabletMetadataPtr> get_tablet_metadata(const std::string& meta_full_path);
```
 Keep a consistent code style.